exe/sufia_export

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ruby
+require 'rails'
+
+# get the current running directory & load the environment from there
+require ::File.join(Dir.pwd, 'config/environment')
```

Comment:
Can you add instructions that clarify that `sufia_export` *must* be run from the application root, else the environment won't be picked up? Either that or use `Rails.root` or the like to find the environment.